.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "runtime"]
+	path = runtime
+	url = git@github.com:apache/camel-k-runtime.git
```

Comment:
That should probably be changed to using the HTTPS protocol.

Reply:
yep, let me retry